### DISCUSSION TITLE
傀影肉鸽一层自动放弃探索（mac m1）以及刷理智时无法输入企鹅物流id

### DISCUSSION THREAD

**Discussion by Aclear211** [MaaAssistantArknights/MaaAssistantArknights#5504] [Q&A]
在自动肉鸽打傀影的时候，在第一层的不期而遇遇到大锁节点会出现选择希望+1后卡顿在结算画面。十数秒后会退出肉鸽并在傀影肉鸽主界面选择放弃探索。（设置选择蜡烛优先，打层数）
第二个问题是，在刷理智的页面无法正确的输入企鹅物流的id或成功输入后，在点击开始任务后企鹅物流id会自动消失。

**Comment by TPL1992** [comment]
我也有这个问题，不期而遇就自动放弃
